market/connector/google-geo-api/google-geo-api-demo/processes/searchLocation.mod

```diff
@@ -0,0 +1,56 @@
+[Ivy]
+178C940D70614226 9.2.0 #module
+>Proto >Proto Collection #zClass
+sn0 searchLocation Big #zClass
```

Comment:
Please separate the re-usable connector parts which can be used in every project, from pure Demo artifacts. 
The docuSign connector may acts as good example:
![docuSignNamesapces](https://user-images.githubusercontent.com/15085693/115832252-f9aa8280-a412-11eb-9287-b3e0cca2cf1c.png)

So applied to your connector:

- [ ] move the Process 'searchLocation.mod' into a 'Demo' namespace Processes/Demo/searchLocation
- [ ] move the DataClass 'google.getp.Data' to 'com.google.map.demo.Data'
- [ ] move the HtmlDialog 'google.getp.map' to 'com.google.map.demo.MapDialog'

Reply:
- [x] move the Process 'searchLocation.mod' into a 'Demo' namespace Processes/Demo/searchLocation
- [x]  move the DataClass 'google.getp.Data' to 'com.google.map.demo.Data'
- [x]  move the HtmlDialog 'google.getp.map' to 'com.google.map.demo.MapDialog'